styles/style.css

```diff
@@ -95,8 +104,38 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+  .button{
+    background-color: var(--orange-dark);
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    border-radius: 3px;
+    border:none;
+    padding: 8px 12px;
+    color:var(--white);
+    text-decoration: none;
+
+}
+  .button:hover {
+    background-color: var(--white);
+    text-align: center;
+    text-decoration: none;
+    border-radius: 3px;
+    padding: 8px 12px;
+    color:var(--orange-dark);
+
+}
+
+  .donation:hover {
```

Comment:
There are some duplicated properties in your button classes. Do you know how you can make this code more DRY (Don't Repeat Yourself)? Let me know if you would like an example :)

Reply:
Your right! I've cleaned up about 7 lines :)